### DISCUSSION TITLE
Memory leak with react-globe.gl

### DISCUSSION THREAD

**Discussion by eFutureDev** [community/community#45719] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I tried to put globe with react-globe.gl on my dashboard and it looked great.
But I found that I am getting memory leak error after long time rendering.
I removed all labels and styles but it's the same result.
This is an urgent matter and hope to discuss asap.

**Comment by iamsuiux** [comment, marked as answer]
Maybe you can upgrade the dependencies.
I upgraded three-globe to 2.24.13 and fixed it in the past.

**Comment by SuperHeroDev** [comment]
In react-globe.gl, you can remove unnecessary labels and countries.
For example, You can filter countries by rank.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @eFutureDev ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
